### PR TITLE
fix(sheets-drawing): drawing snapshot error

### PR DESCRIPTION
**Pull request by weird94**
close #xxx

## Pull Request Checklist

- [ ] Related tickets or issues have been linked in the PR description (or missing issue).
- [ ] [Naming convention](https://github.com/dream-num/univer/blob/dev/docs/NAMING_CONVENTION.md) is followed (**do please** check it especially when you created new plugins, commands and resources).
- [ ] Unit tests have been added for the changes (if applicable).
- [ ] Breaking changes have been documented (or no breaking changes introduced in this PR).

### CODE REVIEW

File: packages/sheets-drawing/src/controllers/sheet-drawing.controller.ts [hunk 1]
```diff
@@ -66,6 +66,7 @@ export class SheetsDrawingLoadController extends Disposable {
                 },
                 onLoad: (unitId, value) => {
                     this._sheetDrawingService.registerDrawingData(unitId, value);
+                    this._drawingManagerService.registerDrawingData(unitId, value);
```
Comment (wzhudev) [inline packages/sheets-drawing/src/controllers/sheet-drawing.controller.ts:69]: Why should we register the value twice on different service? IMHO, `_sheetDrawingService` should do this.

Comment (weird94) [reply packages/sheets-drawing/src/controllers/sheet-drawing.controller.ts:69]: I don't know why, but this was actually 2 service to manager drawing

Comment (weird94) [reply packages/sheets-drawing/src/controllers/sheet-drawing.controller.ts:69]: > Why should we register the value twice on different service? IMHO, `_sheetDrawingService` should do this.

I don't know why, but this was actually 2 service to manager drawing

Comment (weird94) [reply packages/sheets-drawing/src/controllers/sheet-drawing.controller.ts:69]: cc @DR-Univer

Comment (Jocs) [reply packages/sheets-drawing/src/controllers/sheet-drawing.controller.ts:69]: Doc also has two services, but docDrawingService will be deprecated in the future.

**Review by wzhudev — APPROVED** [review]
LGTM

**Pull request merged**